source/ui/beamline/AMXRFDetailedDetectorView.cpp

```diff
@@ -167,6 +172,8 @@ void AMXRFDetailedDetectorView::buildEnergyRangeSpinBoxView()
 {
 	AMSpectrumAndPeriodicTableView::buildEnergyRangeSpinBoxView();
 
+	connect(exportButton_, SIGNAL(clicked()), this, SLOT(onExportButtonClicked()));
```

Comment:
Should this connect statement be in the constructor with the others? Or is it better to keep it here?